ext/readme.md

```diff
@@ -1,6 +1,7 @@
 ## ext
  Status  |  Package  |  Version / Commit  | Usage | Notes
 :------: | :-------: | :----------------: | :---- | :----
+:: | `adba` | pymedusa/[0e1657f](https://github.com/pymedusa/adba/tree/119b9d30a30feb97bcea5e1dc742c82340300091) | **`medusa`** | -
```

Comment:
Change the link text (inside the brackets) to the 7 first characters of the commit SHA (`119b9d3`)